packages/discord.js/src/structures/InteractionCallback.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const { DiscordSnowflake } = require('@sapphire/snowflake');
+
+/**
+ * Represents an interaction callback response from Discord
+ */
+class InteractionCallback {
+  constructor(client, data) {
+    /**
+     * The client that instantiated this.
+     * @name InteractionCallback#client
+     * @type {Client}
+     * @readonly
+     */
+    Object.defineProperty(this, 'client', { value: client });
+
+    /**
+     * The id of the original interaction response
+     * @type {Snowflake}
+     */
+    this.id = data.id;
+
+    /**
+     * The type of the original interaction
+     * @type {InteractionType}
+     */
+    this.type = data.type;
+
+    /**
+     * The id of the channel the original interaction was sent in
+     * @type {?Snowflake}
+     */
+    this.channelId = data.channel_id ?? null;
+
+    /**
+     * The id of the guild the original interaction was sent in
+     * @type {Snowflake}
+     */
+    this.guildId = data.guild_id ?? null;
```

Comment:
These properties appear to be undocumented. I believe they should be removed, if I'm not mistaken.
https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-callback-interaction-callback-object